src/targets/browser/index.ejs

```diff
@@ -1,32 +1,45 @@
 <!DOCTYPE html>
 <html lang="{{.Locale}}">
-<head>
-  <meta charset="utf-8">
-  <title><%= htmlWebpackPlugin.options.title %></title>
-  <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
-  <link rel="icon" type="image/png" href="./favicon-32x32.png" sizes="32x32">
-  <link rel="icon" type="image/png" href="./favicon-16x16.png" sizes="16x16">
-  <link rel="manifest" href="/manifest.json" crossorigin="use-credentials">
-  <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#297EF2">
-  <% if (__TARGET__ !== 'mobile') { %>
-  <link rel="stylesheet" type="text/css" href="//{{.Domain}}/assets/fonts/fonts.css">
-  <% } %>
-  <meta name="theme-color" content="#ffffff">
-  <meta name="viewport" content="width=device-width, height=device-height, initial-scale=1, viewport-fit=cover">
-  <% _.forEach(htmlWebpackPlugin.files.css, function(file) { %>
-      <link rel="stylesheet" href="<%- file %>">
-  <% }); %>
-  <% _.forEach(htmlWebpackPlugin.files.js, function(file) { %>
-      <script src="<%- file %>" defer></script>
-  <% }); %>
-  <% if (__TARGET__ === 'mobile') { %>
-  <meta name="format-detection" content="telephone=no">
-  <script src="cordova.js" defer></script>
-  <% } else if (__STACK_ASSETS__) { %>
-  {{.CozyBar}}
-  <% } %>
-</head>
-<div
-  role="application"
-  data-cozy="{{.CozyData}}"
->
+  <head>
+    <meta charset="utf-8" />
+    <title><%= htmlWebpackPlugin.options.title %></title>
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
+    <link
+      rel="icon"
+      type="image/png"
+      href="./favicon-32x32.png"
+      sizes="32x32"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      href="./favicon-16x16.png"
+      sizes="16x16"
+    />
+    <link rel="manifest" href="/manifest.json" crossorigin="use-credentials" />
+    <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#297EF2" />
+    <% if (__TARGET__ !== 'mobile') { %>
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="//{{.Domain}}/assets/fonts/fonts.css"
+    />
+    <% } %>
+    <meta name="theme-color" content="#ffffff" />
+    <meta
+      name="viewport"
+      content="width=device-width, height=device-height, initial-scale=1, viewport-fit=cover"
+    />
+    <% _.forEach(htmlWebpackPlugin.files.css, function(file) { %>
+    <link rel="stylesheet" href="<%- file %>" />
+    <% }); %> <% _.forEach(htmlWebpackPlugin.files.js, function(file) { %>
+    <script src="<%- file %>" defer></script>
+    <% }); %> <% if (__TARGET__ === 'mobile') { %>
+    <meta name="format-detection" content="telephone=no" />
+    <script src="cordova.js" defer></script>
+    <% } else { %>
```

Comment:
this is the only non-prettier change here

Reply:
Why did you remove the __STACK_ASSETS__ test? 

Since cozy-bar is still in 7.X, I don't think there is a BC somewhere

Reply:
Somehow it prevented the bar from loading, not sure why?

Reply:
This is not the right fix then. 

What is the error when you revert it?

Reply:
window.cozy.bar === undefined

Reply:
maybe it was a side-effect of cozy-flags, let me try again

Reply:
![image](https://user-images.githubusercontent.com/12577784/128475284-517e8324-2daa-4d7e-9c1b-b4c63e418d6e.png)

nope still broken even after having fixed cozy-flags

Reply:
linked to this? 
![image](https://user-images.githubusercontent.com/12577784/128476183-82300bb3-5846-4cd2-bade-23bdc6eab92c.png)